server/app/views/admin/programs/ManageProgramAdminsView.java

```diff
@@ -64,72 +71,105 @@ public Content render(
             .setTitle(fullTitle)
             .addMainContent(
                 renderHeader(fullTitle),
-                adminEmailTemplate(),
-                renderAdminForm(request, program.id(), existingAdminEmails));
+                renderHeader("Add New Admin"),
+                renderAddNewButton(request, program.id()),
+                renderHeader("Existing Admins"),
+                renderExistingAdmins(request, program.id(), existingAdminEmails),
+                renderBackButton());
 
     message.map(m -> m.setDuration(6000)).ifPresent(htmlBundle::addToastMessages);
 
     return layout.renderCentered(htmlBundle);
   }
 
-  /**
-   * Render a form with inputs for program admin emails. If program admins exist, the input fields
-   * will be pre-populated with their email addresses.
-   */
-  private FormTag renderAdminForm(
+  private DivTag renderAddNewButton(Http.Request request, long programId) {
```

Comment:
Maybe renderAddNewAdminForm rather than Button since it returns the whole form and not just the button?

Reply:
Done